apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -1053,7 +1056,7 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
                         defaultActiveIndex={ defaultActiveIndex }
                         onTabChange={ handleTabChange }
                         panes={ resolveTabPanes() }
-                        onInitialize={ ({ panesLength }) => {
+                        onInitialize={ ({ panesLength }: { panesLength: number }) => {
```

Comment:
Since this is defined in the https://github.com/wso2/identity-apps/blob/master/modules/react-components/src/components/tab/resource-tab/resource-tab.tsx#L62 like this. we can keep this as it is